app/assets/stylesheets/groups_form.css

```diff
@@ -0,0 +1,45 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 30px;
+}
+
+.field,
+.field-two {
+  padding-bottom: 20px;
+}
+
+.field label {
+  padding-right: 10px;
+  font-size: 20px;
+}
+
+.field-two label {
+  padding-right: 21px;
+  font-size: 20px;
+}
+
+.field-three {
+  margin-top: 20px;
+  width: 100%;
+  border: solid 1px #60b625;
+  height: 60px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #60b625;
+  padding: 20px 0;
+  justify-self: flex-end;
+}
+
+.button {
+  padding: 20px;
+  text-decoration: none;
+  color: white;
+  font-size: 25px;
+  border: solid 1px white;
+  width: 84%;
+  text-align: center;
+  background-color: #60b625;
+}
```

Comment:
- [ ] [OPTIONAL]
Let's add back arrow button in the navigation to enable users to able to the previous page.
- [ ] [OPTIONAL]
Similarly, let's ensure the `Add new Category` and similar buttons are **always** sticky to the bottom of the mobile screen, like seen in the original design. Let's also ensure the same for the similar other pages.

**Your design:**
**Your splash screen:**     |  **Expected**     
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/shm04/budget-app/assets/12745474/588dd208-bc97-4905-a8a6-61ddf11cb84f"> | <img width="800" alt="Home Page" src="https://github.com/M-Anwar-Hussaini/Expense-Management/assets/12745474/b23a42bf-ee8c-46d6-8ca9-f0b762971127">